libs/ui-lib/lib/common/components/CustomManifests/DeleteCustomManifestModal.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
```

Comment:
we will need i18n here

Reply:
Can you also please make sure that `onDelete` is a Promise returning function.
The loading and error states are handled